CMakeLists.txt

```diff
@@ -154,6 +154,9 @@ endif()
 set(SDL_LIBC_DEFAULT ON)
 set(SDL_SYSTEM_ICONV_DEFAULT ON)
 if(WINDOWS)
+  if(MINGW)
+    set(SDL_LIBC_DEFAULT OFF)
+  endif()
   set(SDL_SYSTEM_ICONV_DEFAULT OFF)
```

Comment:
@madebr, it's just pow() that's failing automated testing. Is there a way we can blacklist just that function, or maybe it's just a bug in the runtime that's already been fixed?

Reply:
The error message reads
```
Pow(-72.300000,12.000000), expected [20401381050275984310272.000000], got 20401381050275996893184.000000'}
```
With Python, I calculated the exact number to `-20401381050275994437455.127797667121`.
So the mingw error is smaller then the msvc error.

So we whould do the pow test with an inexact comparison instead of an exact one.

Reply:
The inexact comparison also fails, because it checks within EPSILON, but this is a large difference. It's worth noting that the expected value passes for all platforms except MINGW. Maybe we should just disable that particular test?